### PR TITLE
US124842/add mobile overflow menu button.

### PR DESCRIPTION
**Pull request by mbosiljevac**
Note: the more options button is shifted over to keep the margins consistent.
image

### CODE REVIEW

File: components/right-panel/consistent-evaluation-right-panel.js [hunk 1]
```diff
@@ -144,6 +149,31 @@ export class ConsistentEvaluationRightPanel extends LocalizeConsistentEvaluation
 		this.canRecordFeedbackAudio = false;
 		this.useNewHtmlEditor = false;
 		this.rubricsOpen = 0;
+		this.mobileMediaQuery = window.matchMedia('(max-width: 767px)');
```
Comment (vix633) [inline components/right-panel/consistent-evaluation-right-panel.js:152]: I don't actually know enough about media queries to review this. @AlexBedley could you peek at this?

**Review by jeroach — COMMENTED** [review]
I have a feeling that we should be handling the mobile menu via css media queries alone.

e.g.
- different class names on the default menu and mobile menu
- on standard screen size, default is visible, mobile is hidden
- once the screen width is small enough, hide default show mobile

Thoughts?

File: components/right-panel/consistent-evaluation-right-panel.js (hunk 1 above)
Comment (jeroach) [reply components/right-panel/consistent-evaluation-right-panel.js:152]: Yeah I don't think this is the approach we want to take. See my larger comment.

**Comment by mbosiljevac** [comment]
> I have a feeling that we should be handling the mobile menu via css media queries alone.
> 
> e.g.
> 
> * different class names on the default menu and mobile menu
> * on standard screen size, default is visible, mobile is hidden
> * once the screen width is small enough, hide default show mobile
> 
> Thoughts?

makes sense! I'll play around with that idea and get something working :)

File: components/right-panel/consistent-evaluation-right-panel.js [hunk 2]
```diff
@@ -105,6 +105,23 @@ export class ConsistentEvaluationRightPanel extends LocalizeConsistentEvaluation
 
 	static get styles() {
 		return  css`
+			.d2l-consistent-evaluation-right-panel-overflow-menu-mobile {
+				display: none;
+			}
+
+			@media (max-width: 767px) {
```
Comment (JamieTran) [inline components/right-panel/consistent-evaluation-right-panel.js:112]: nit: We can group classes together that have the same `@media` queries
eg:

```
@media (max-width: 767px) {
		.d2l-consistent-evaluation-right-panel-overflow-menu-mobile {
			display: inline-block;
			margin: 0.5rem 1.25rem 0.5rem 1.25rem;
		}
		.d2l-consistent-evaluation-right-panel-clearfix {
			display: none;
		}
	}
```

**Review by JamieTran — APPROVED** [review]

**Pull request merged**